conan/tools/build/cross_building.py

```diff
@@ -28,3 +28,14 @@ def get_cross_building_settings(conanfile):
                 os_host, arch_host)
     else:
         return os_host, arch_host, os_host, arch_host
+
+
+def can_run(conanfile):
+    """
+    Validates if the current build platform can run a file which is not for same arch
+    See https://github.com/conan-io/conan/issues/11035
+    """
+    allowed = conanfile.conf.get("tools.build.cross_building:can_run", check_type=bool)
+    if not allowed:
```

Comment:
I would take both ``True`` and ``False`` as definitive values, and return them.

And then, only fallback to ``not cross_building()`` for the case the ``can_run`` is ``None`` (not defined by user)

Reply:
Yes, totally makes sense! I gonna update it!

Reply:
Done.